main.go

```diff
@@ -76,7 +76,7 @@ func registerSIGINTHandler(mountPoint string) {
 }
 
 func getUserAgent(appName string) string {
-	return strings.TrimSpace(fmt.Sprintf("gcsfuse/%s %s %s", getVersion(), appName, os.Getenv("GCSFUSE_METADATA_IMAGE_TYPE")))
+	return strings.TrimSpace(fmt.Sprintf("gcsfuse/%s %s %s", getVersion(), appName, os.Getenv("GCSFUSE_METADATA_IMAGE_TYPE"))) + " (GPN: gcsfuse)"
```

Comment:
can you add it as part of "gcsfuse/%s %s %s GPN: gcsfuse" instead of doing another append

Reply:
Tried doing this initially but TestGetUserAgentWhenMetadataImageTypeEnvVarIsNotSet was failing because of an extra space for empty environment variable. I've updated the logic to remove any redundant spaces.